File: AssetManagement/AssetManagement.Web/Modules/AddAnAsset/AssetDetails/DepreciableAsset.cs
using Serenity.ComponentModel;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Web;

namespace AssetManagement
{
    [EnumKey("AssetDetails.DepreciableAsset"), ScriptInclude]
    public enum DepreciableAsset
    {
        [Description("No")]
        No = 1 ,
        [Description("Yes")]
        Yes = 2 
    }
}

File: AssetManagement/AssetManagement.Web/Imports/ServerTypings/DepreciableAsset.ts
namespace AssetManagement {
    export enum DepreciableAsset {
        No = 1,
        Yes = 2
    }
    Serenity.Decorators.registerEnumType(DepreciableAsset, 'AssetManagement.DepreciableAsset', 'AssetDetails.DepreciableAsset');
}

